src/main/resources/graphql/team.graphqls

```diff
@@ -2,15 +2,31 @@ type Team {
     id: ID!,
     abbreviation: String!,
     city: String!,
-    conference: String!,
-    division: String!,
+    conference: Conference!,
+    division: Division!,
     fullName: String!,
     name: String!,
     players: [Player]!
 }
 
 type Query {
-    # Give -1 to get all teams.
-    teams(count: Int): [Team]
-    team(id: ID): Team
+    # count: the desired number of teams. -1: all of the teams.
```

Comment:
Instead of using `-1` as a placeholder for unlimited count, why not just omit the `count` parameter? :)